ftw/avatar/tests/test_default_generator.py

```diff
@@ -51,7 +51,7 @@ def test_foreground_color_is_white_by_default(self):
     def test_background_color_is_valid_color_tuple(self):
         self.assertEquals(
             [int, int, int],
-            map(type, DefaultAvatarGenerator().background_color()))
+            [item for item in map(type, DefaultAvatarGenerator().background_color())])
```

Comment:
`map` returns a "mapobject type at 0xSSSSSSSS" item, so we need to unfold the map to a list to compare it in python3

Reply:
Instead of using a list-comprehension I would just use a simple `list(...)`.
```suggestion
            list(map(type, DefaultAvatarGenerator().background_color()))
```

Reply:
Indeed 🤦